serializers/custom_wizard/wizard_serializer.rb

```diff
@@ -12,6 +13,7 @@ class CustomWizard::WizardSerializer < CustomWizard::BasicWizardSerializer
              :categories
 
   has_many :steps, serializer: ::CustomWizard::StepSerializer, embed: :objects
+  has_one :first_step, serializer: ::CustomWizard::StepSerializer, embed: :objects
```

Comment:
@fzngagan isn't the first step already in ``steps`` why do we need to serialize this seperately?